R/compare2.R

```diff
@@ -0,0 +1,68 @@
+sqrt_comparison <- function(l_unique, l_total){
```

Comment:
Add Docstring at top of these files to define What TRUE and FALSE mean in terms of data quality.